### DISCUSSION TITLE
How do I contect to someone for Help

### DISCUSSION THREAD

**Discussion by FsAguard62** [community/community#23892] [New to GitHub]
<p>Well, I have the browser extension named “Copy Plain Text” for Firefox and it doesn’t work on the webpage I am using it for.</p>
<p>This is the webpage: <a href="https://cncapp04.alliances.commandandconquer.com/425/index.aspx?langPreferred=en_US" rel="noopener nofollow ugc">https://cncapp04.alliances.commandandconquer.com/425/index.aspx?langPreferred=en_US</a></p>
<p>I Doesn’t work in the Chat and Message portions of this site.</p>
<p>This website in some cases doesn’t let you use the Left Click at all!</p>
<p>Looking For Some help.</p>

**Comment by Zerotask** [comment]
<p>Then you have to look for a specific repository or other developer contact information on the plugin page.</p>
<p>This forum is for git and GitHub related support.</p>

**Comment by FsAguard62** [comment]
<p>He is on here and there is no other info and support on the app itself nor the Firefox add-on extension webpage this is place was the only place that I found on the options page for the app was the link to his list of updates on here.</p>
<p>This is the page that had found: <a href="https://github.com/erosman/support/" rel="noopener nofollow ugc">https://github.com/erosman/support/</a>  but there’s no way to contact him from here</p>
<p>I am new to this stuff and only the ones that dealt with on here is the AdGuard Team</p>

**Comment by airtower-luna** [comment, marked as answer]
<aside class="quote no-group" data-username="FsAguard62" data-post="3" data-topic="199344">
<div class="title">
<div class="quote-controls"></div>
 FsAguard62:</div>
<blockquote>
<p>but there’s no way to contact him from here</p>
</blockquote>
</aside>
<p>There is, to quote the README in that repository:</p>
<blockquote>
<p>Please post all feature requests and issue to “Issues”.</p>
</blockquote>
<p>Click on “issues”, use the search to see if your problem has already been reported, and if not use the “new issue” button. :slightly_smiling_face:</p>
